tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -695,10 +700,32 @@ public:
                             theVariationName);
    }
 
-   /// \brief Register systematic variations for an existing columns using auto-generated variation tags.
-   /// This overload of Vary takes a nVariations parameter instead of a list of tag names. Tag names
-   /// will be auto-generated as the sequence 0...nVariations-1.
-   /// See the documentation of the previous overload for more information.
+   /// \brief Register systematic variations for a single existing column using auto-generated variation tags.
+   /// \param[in] colName set of names of columns for which varied values are provided.
+   /// \param[in] expression a callable that evaluates the varied values for the specified columns. The callable can
+   ///            take any column values as input, similarly to what happens during Filter and Define calls. It must
+   ///            return an RVec of varied values, one for each variation tag, in the same order as the tags.
+   /// \param[in] inputColumns the names of the columns to be passed to the callable.
+   /// \param[in] nVariations number of variations returned by the expression. The corresponding tags will be "0", "1",
+   /// etc. \param[in] variationName a generic name for this set of varied values, e.g. "ptvariation".
+   ///            colName is used if none is provided.
+   ///
+   /// This overload of Vary takes an nVariations parameter instead of a list of tag names. Tag names
```

Comment:
Maybe add another bit of explanation about the `variationName` (very rough probably can be improved):
"The varied results will be accessible via the keys of the dictionary with the form `variationName:N` where N is the corresponding sequential tag up to `nVariations`"